hercules-ci.nix

```diff
@@ -17,5 +17,5 @@
     };
   };
 
-  herculesCI.ciSystems = [ "x86_64-linux" ];
+  herculesCI.ciSystems = [ "x86_64-linux" ]; # TODO(szg251): fix darwin CI build and add "x86_64-darwin"
```

Comment:
I wanted to enable darwin, because it works on my machine.
Funnily (yeah, it wasn't that funny) I could build and run tests on a local x86_64-darwin machine, but it fails on CI.
It looks like there's an upstream issue with plutip and ogmios